integration-tests/jpa/src/main/java/io/quarkus/it/jpa/preferredhibernatetypesoverride/OverriddenPreferredTypesResource.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.it.jpa.preferredhibernatetypesoverride;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.hibernate.Session;
+import org.hibernate.type.StandardBasicTypes;
+import org.jboss.resteasy.reactive.RestQuery;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Path("/overridden-preferred-types")
+@ApplicationScoped
+public class OverriddenPreferredTypesResource {
+
+    @Inject
+    Session session;
+
+    @GET
+    @Path("/test")
+    @Produces(MediaType.TEXT_PLAIN)
+    @Transactional
+    public String test(@RestQuery String expectedSchema) {
+        assertThat(findUsingNativeQuery(expectedSchema)).isEmpty();
```

Comment:
No need to customize the schema, it's not relevant to this test.

```suggestion
    public String test() {
        assertThat(findUsingNativeQuery()).isEmpty();
```